warn/warn_docstring_test.go

```diff
@@ -322,6 +322,20 @@ def f(x, y, z = None, *args, **kwargs):
 		[]string{},
 		scopeEverywhere)
 
+	checkFindings(t, "function-docstring-args", `
+def f(x, *, y, z = None):
+   """This is a function.
+
+   Args:
+    x: x
+    y: y
+    z: z
+   """
+   pass
+`,
+		[]string{},
+		scopeEverywhere)
+
```

Comment:
Add a negative test, to check the error message when one argument is *

Reply:
Done.